students/jason_jenkins/lesson04/mailroom.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+
+"""
+Lesson 3: Mail Room Part 1
+Course: UW PY210
+Author: Jason Jenkins
+"""
+import pathlib
+
+
+# Global Variables
+run_program = True
+donor_dict = dict()
+
+
+def send_thanks():
+    """
+    Method used to probt donor name or list out donors
+    """
+
+    global donor_dict
+
+    response = ""
+
+    while(True):
+        response = input('Input donors full name, "list", or "exit": ')
```

Comment:
use lower() with the input function to avoid duplicating -> `response = input("...").lower()`